lib/server.js

```diff
@@ -1169,5 +1170,54 @@ function _finishReqResCycle(req, res, route, err) {
     }
 
     // decrement number of requests
-    self._unfinishedRequests--;
+    self._inflightRequests--;
+};
+
+
+
+/**
+ * Return debug information about the server.
+ * @public
+ * @method debugInfo
+ * @returns {Object}
+ */
+Server.prototype.debugInfo = function debugInfo() {
```

Comment:
Oops sorry, should have removed this comment after I understood what was going on. This is what `toJSON()` would call. I think `debugInfo()` is fine.